aviary/aviary.py

```diff
@@ -561,6 +561,23 @@ def main():
         default=200000
     )
 
+    binning_group.add_argument(
+        '--coverage-job-strategy', '--coverage_job_strategy',
+        help=f'Strategy for splitting samples when calculating coverage for binning. Choices include: \n'
```

Comment:
Kinda sounds like individual samples are split. Maybe start with a problem, then solution. Like, when large numbers of samples are used for cobinning, it can be more computationally scalable to calculate coverage split up across jobs, etc..

Reply:
Reworded